Density/app/src/main/java/org/cornelldti/density/density/util/ColorBarMarkerView.kt

```diff
@@ -15,6 +15,7 @@ import com.github.mikephil.charting.data.Entry
 import com.github.mikephil.charting.highlight.Highlight
 
 import org.cornelldti.density.density.R
+import java.util.*
```

Comment:
This change looks extra

Reply:
True, I'm not sure why it was added. I will try fixing and building again.

Reply:
Ok, it's used for `Date`, `Calendar`, and `Locale` so I've just imported them on their own. It's reflected in my latest PR.